src/events/ready.js

```diff
@@ -20,24 +18,21 @@ class Ready {
 				type: 'WATCHING',
 			} ];
 
-		this.client.logger.info('Loading Slash Commands...');
-		const table = new AsciiTable('Slash Commands');
-		table.setHeading('Command Name', 'Type', 'Status');
+		// this.client.logger.info('Loading Slash Commands...');
 
-		const commandArray = [];
+		// const commandArray = [];
 
-		this.client.commands.each(async command => {
-			if(command.disabled) return table.addRow(command.name, this.client.utils.capitalize(command.type), 'Fail');
-			if(command.ownerOnly || command.type === this.client.types.OWNER) return table.addRow(command.name, this.client.utils.capitalize(command.type), 'Fail');
+		// this.client.commands.each(async command => {
+		// 	if(command.disabled) return table.addRow(command.name, this.client.utils.capitalize(command.type), 'Fail');
+		// 	if(command.ownerOnly || command.type === this.client.types.OWNER) return table.addRow(command.name, this.client.utils.capitalize(command.type), 'Fail');
 
-			commandArray.push(command.generateSlashCommand());
+		// 	commandArray.push(command.generateSlashCommand());
 
-			table.addRow(command.name, this.client.utils.capitalize(command.type), 'Pass');
-		});
+		// 	table.addRow(command.name, this.client.utils.capitalize(command.type), 'Pass');
+		// });
 
 		// const test_guild = await this.client.guilds.fetch('789215359878168586');
 		// test_guild.commands.set(commandArray);
-		this.client.logger.log(table.toString());
 
 		this.client.user.setPresence({
```

Comment:
Can we replace this with a fixed status as per discord developer TOS

Reply:
👍

Reply:
Oops, selected wrong line, see the lines below